src/sagemaker/remote_function/client.py

```diff
@@ -185,6 +185,7 @@ def remote(
           methods that are not available via PyPI or conda. Default value is ``False``.
 
         instance_count (int): The number of instances to use. Defaults to 1.
+          Remote function do not support instance_count > 1
```

Comment:
suggestion framing the statement as `NOTE: Remote function does not support . . . `